views/forms/ResetPasswordForm/index.jsx

```diff
@@ -0,0 +1,67 @@
+import Form from '../Form'
+import TextInput from '../TextInput'
+import { Button } from '@material-ui/core'
+
+const ResetPasswordForm = ({
+  classes,
+  onSubmit,
+  control,
+  errors,
+  navigate,
+}) => {
+  return (
+    <Form onSubmit={onSubmit}>
+      <TextInput
+        control={control}
+        name="username"
+        label="Username"
+        required={true}
+        margin="normal"
+      />
+      <TextInput
+        control={control}
+        name="password"
+        label="Password"
+        required={true}
+        margin="normal"
+        error={errors.password.error}
+        helperText={errors.password.message}
+      />
+      <TextInput
+        control={control}
+        name="confirmPassword"
```

Comment:
looks like we're mixing cases with the names of fields (`confirmPassword` and `reset_key`, for example). I think we should decide on the casing to use and be consistent in this form and elsewhere.

Reply:
I feel like this change would lead to a regression, the reset_key property is what they use in the db. I think we should take care of this down the line as this change would impact current users who have requested to have their password reset.